cpp/ql/src/experimental/Security/CWE/CWE-670/DangerousUseSSL_shutdown.cpp

```diff
@@ -0,0 +1,11 @@
+...
+SSL_shutdown(ssl); 
+SSL_shutdown(ssl); // BAD
+...
+    switch ((ret = SSL_shutdown(ssl))) {
```

Comment:
```suggestion
    switch (ret = SSL_shutdown(ssl)) {
```

Reply:
That extra set of parentheses might actually be intentional. It's the standard way of telling people that "yes, I know that this is an assignment in a conditional". We even have a [query](https://github.com/github/codeql/blob/main/cpp/ql/src/Likely%20Bugs/Likely%20Typos/AssignWhereCompareMeant.ql#L55) that uses to weed out false positives 😄.

Reply:
Ah, I had forgotten about that pattern.  Please disregard my suggestion.